docs/introduction/faq-and-common-issues.md

```diff
@@ -125,4 +126,7 @@ Otherwise, the logic is in facades (resp. the facades are used as delegates to o
 See [Friendly URL](/docs/introduction/friendly-url.md) article.
 
 ## How can I create Front-end Breadcrumb navigation?
-See [Front-end Breadcrumb Navigation](/docs/introduction/front-end-breadcrumb-navigation.md) article.
\ No newline at end of file
+See [Front-end Breadcrumb Navigation](/docs/introduction/front-end-breadcrumb-navigation.md) article.
+
+## Is there any way how to see the progress of Codeception acceptance tests when using Docker?
```

Comment:
"... progress of Codeception acceptance tests ..." at first I thought about some progress bars while running.
I don't know better phrasing, but currently, it's not propagating the awesomeness enough 😄  
(not a CR just comment)

Reply:
it took me quite a long time to come up with these words :laughing: another idea was something like this
```
Is there any way how to see what is going on in the Codeception acceptance tests when using Docker?
```

Reply:
What about

`Can I see what is really happening in the Codeception acceptance tests when using Docker?`

Reply:
rewritten :wink: